index.bs

```diff
@@ -423,6 +418,16 @@ To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [
         error-handling behavior.
 
 
+Integration with Service Worker {#integration-sw}
+-------------------------------------------------
+
+In https://w3c.github.io/ServiceWorker/#dom-fetchevent-respondwith, insert the following item
+after 10.1.
+
+2. Else if the result of performing a [$cross-origin resource policy check$] with _event_'s
+request's associated request and _response_'s associated response is `blocked`, then set
+the respond-with-error flag.
```

Comment:
Nit: I think I'd fold this into step 10.1. That is "If |response| is not a `Response` object, or if the result of performing a [$ ... $] ... is `blocked`, then set the respond-with-error flag."

WDYT?

Reply:
SG, fixed. thank you.